suite/tests/linux/clone.c

```diff
@@ -245,10 +245,6 @@ make_clone3_syscall(struct clone_args *clone_args, uint clone_args_size,
 #        endif
 #    elif defined(AARCH64)
     uint64 clone_args_size_64 = (uint64)clone_args_size;
-    /* Do not add code between above declarations and their use below.
```

Comment:
Found this stray comment. It's unrelated to this PR, but I thought I'll clean it up.